public/js/map.js

```diff
@@ -223,25 +223,37 @@ const map = {
       maxZoom: 7,
       gridSize: 65,
       styles: [
-        {
-          width: 30,
+        MarkerClusterer.withDefaultStyle({
+          url: "/images/map-marker-images/m3.svg",
+          width: 20,
           height: 30,
-          className: "custom-clustericon-1",
-        },
-        {
-          width: 40,
-          height: 40,
-          className: "custom-clustericon-2",
-        },
-        {
+          anchorText: [-3, 0],
+          anchorIcon: [30, 10],
+          textSize: 11,
+          textColor: "#fff",
+        }),
+        MarkerClusterer.withDefaultStyle({
+          url: "/images/map-marker-images/m4.svg",
+          width: 35,
+          height: 53,
+          anchorText: [-5, 0],
+          anchorIcon: [53, 17],
+          textColor: "#fff",
+          textSize: 13,
+        }),
+        MarkerClusterer.withDefaultStyle({
+          url: "/images/map-marker-images/m5.svg",
           width: 50,
-          height: 50,
-          className: "custom-clustericon-3",
-        },
+          height: 75,
+          anchorText: [-10, 0],
+          anchorIcon: [75, 25],
+          textSize: 14,
+          textColor: "#fff",
```

Comment:
@ascott Should we make this `textColor` a variable? They're all duplicates of white. In case we need to change it in the future, we can just change it in a single place.